sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/DeduplicateRelations.scala

```diff
@@ -227,7 +227,7 @@ object DeduplicateRelations extends Rule[LogicalPlan] {
         newVersion.copyTagsFrom(oldVersion)
         Seq((oldVersion, newVersion))
 
-      case oldVersion @ FlatMapCoGroupsInPandas(_, _, _, output, _, _)
+      case oldVersion@FlatMapCoGroupsInPandas(_, _, output, _)
```

Comment:
convention is spaces around `@`:
```suggestion
      case oldVersion @ FlatMapCoGroupsInPandas(_, _, output, _)
```